src/relay/qnn/util.h

```diff
@@ -0,0 +1,139 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+/*!
+ * \file tvm/relay/quantize_util.h
+ * \brief Utility methods needs for quantized ops that can be shared
+ */
+
+#ifndef TVM_RELAY_QUANTIZE_UTIL_H_
+#define TVM_RELAY_QUANTIZE_UTIL_H_
+
+#include <tvm/expr.h>
+#include <limits>
+#include <tvm/relay/expr.h>
+
+namespace tvm {
+namespace relay {
+
+inline bool IsInt8(const DataType& dtype) {
```

Comment:
This is certainly a too small function to introduce another level of indirection. Directly remove it and keep things in the original condition